charts/ingress-monitor-controller-endpoint/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+name: ingress-monitor-controller-endpoint
+description: A Helm chart to install Ingress Monitor Controller endpoints
+version: 0.1.0
+keywords:
+  - IngressMonitorController
+  - kubernetes
+maintainers:
+- name: Luis Gallardo
+  email: luis.gallardo@binbash.com.ar
```

Comment:
@lgallard Luis let's also add here leverage@binbash.com.ar to the maintainers list please, which is a distribution list that include all Leverage maintainers (you're also included here).

```
maintainers:
- name: Binbash Leverage Team 
  email: leverage@binbash.com.ar
- name: Luis Gallardo
  email: luis.gallardo@binbash.com.ar
 ```

Reply:
Done!